core/rec-common/rec_common_helper.cpp

```diff
@@ -0,0 +1,9 @@
+#include "types.h"
+
+unsigned int ngen_required = true;
```

Comment:
How would this work with the arm JITs? We can't reference global symbols there because of ... linking issues.